modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -10857,6 +10853,13 @@ type TargetEnvironment {
   When set, overrides the default base position of the target group
   """
   explicitBase: Coordinates
+
+  """
+  The name of the guide target, if any, set by `setGuideTargetName`.
+  If the name is no longer valid or a sequence cannot be generated, null will
+  be returned.
+  """
```

Comment:
That it can disappear after having been set could be disquieting to a sensitive person.  I wonder if we should keep it and just issue warnings / errors when the guide environment is sought.

Reply:
I think Explore needs to know when it is invalid. So that it can choose the best one of the (possibly) new choices.

Reply:
Navigate will get the errors.